src/stats/logger.go

```diff
@@ -0,0 +1,71 @@
+package stats
```

Comment:
It seems a shame to copy this entire file. I don't write go anymore, and I doubt gostats is maintained anymore, but is there any way to avoid doing this? Can we just use a much simpler format for this and write directly to our logging sync? I don't think the format needs to stay the same.

Reply:
I agree if the library is not properly maintained we should probably stop using it, but that would be a much more involved change, as we should then consider not only the stdout output.

For the purpose of this PR of removing the noise on the stdout, I've updated the implementation to be very minimal, and in a way that allows us to change the format more easily to anything we consider more appropriate, but keeping the current JSON for now. WDYT?